CHANGELOG.md

```diff
@@ -6,6 +6,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.0] Hyperion - 2021-11-08 [:warning:]
+Reorg-resistance for the Ethereum backend and support for external Watchtowers.
+
+### Added
+
+- :sparkles: **Ethereum backend: Reorg resistance** [#19](https://github.com/hyperledger-labs/go-perun/issues/19): The Ethereum backend now lets the user specify after how many blocks a transaction or an event should be considered confirmed.
+- :sparkles: **Watcher interface** [#172](https://github.com/hyperledger-labs/go-perun/issues/172): The watcher logic is now injectable. The adjudicator takes a watcher instance as a setup parameter. This enables using remote watcher services. A local watcher implementation is provided.
```

Comment:
```suggestion
- :sparkles: **Ethereum backend: Reorg resistance** (#19): The Ethereum backend now lets the user specify after how many blocks a transaction or an event should be considered confirmed.
- :sparkles: **Watcher interface** (#172): The watcher logic is now injectable. The adjudicator takes a watcher instance as a setup parameter. This enables using remote watcher services. A local watcher implementation is provided.
```
Github automatically translates issue and PR numbers into the correct links.

Reply:
In some locations GitHub does this, but apparently not when viewing the [CHANGELOG.md](https://github.com/perun-network/go-perun/blob/f57bff55dcf119a85cd0f5766ebf681b8078240d/CHANGELOG.md).

Reply:
It doesn't 🙈